js/gated-content/src/main.js

```diff
@@ -45,6 +48,12 @@ new Vue({
   mounted() {
     const app = this;
 
+    const cookieName = 'Drupal_visitor_gc_auth_destination';
```

Comment:
Let's rename `Drupal_visitor_gc_auth_destination` to `openy_gc_auth_destination`